src/Microsoft.ApplicationInsights.AspNetCore/Extensibility/Implementation/Tracing/AspNetEventSource.cs

```diff
@@ -140,6 +140,12 @@ public void UnableToFindModuleToConfigure(string moduleType, string appDomainNam
             this.WriteEvent(11, moduleType, this.ApplicationName);
         }
 
+        [Event(12, Message = "Unable to find expected module {0} in service collection.", Level = EventLevel.Warning, Keywords = Keywords.Diagnostics)]
+        public void UnableToFindExpectedModule(string moduleType, string appDomainName = "Incorrect")
```

Comment:
Can you make this message more specific. So customer will know that without this QuickPulse wouldn't work. Generic message doesn't help.

Also promote it to `Error` so support of customer screwed up config and not able to see live metrics would be easier

Reply:
done